src/handlers/featureFlags.js

```diff
@@ -0,0 +1,16 @@
+const handler = (router, routesContext) => {
+    router.get('/featureflags', async (ctx, next) => {
+        let flags = {};
```

Comment:
```suggestion
        const flags = (routesContext.config && routesContext.config.featureFlags)
            ? routesContext.config.featureFlags
            : {};
```

Reply:
ternary operator is often deemed less readable than an ol' fashioned if/then.